cmd/eyescli/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	keycmd "github.com/tendermint/go-crypto/cmd"
+	"github.com/tendermint/tmlibs/cli"
+
+	"github.com/tendermint/basecoin/client/commands"
+	"github.com/tendermint/basecoin/client/commands/auto"
+	"github.com/tendermint/basecoin/client/commands/proxy"
+	"github.com/tendermint/basecoin/client/commands/query"
+	rpccmd "github.com/tendermint/basecoin/client/commands/rpc"
+	"github.com/tendermint/basecoin/client/commands/seeds"
+	txcmd "github.com/tendermint/basecoin/client/commands/txs"
+	etccmd "github.com/tendermint/basecoin/modules/etc/commands"
+)
+
+// EyesCli - main basecoin client command
+var EyesCli = &cobra.Command{
+	Use:   "eyescli",
+	Short: "Light client for tendermint",
```

Comment:
Capitalize Tendermint (as per coding standards) - should only be lowercase ever when it's a part of the cli-command name (`Use`) or file/folder name